doc/thingpedia-tutorial-hello-world.md

```diff
@@ -0,0 +1,80 @@
+# Tutorial 0: Hello World
+
+Welcome to Almond! This tutorial will get you started making your very first Almond device. 
```

Comment:
Is the expectation that the user already has a developer account in this tutorial? Is that described elsewhere?

Reply:
That's on here right? https://almond-dev.stanford.edu/doc/getting-started.md

Reply:
Added a line to direct users to Getting Started if they don't have a developer account.